cmd/makedocs/main.go

```diff
@@ -56,3 +58,10 @@ func getDesc(name string) (string, error) {
 	b, err := ioutil.ReadFile(checkersPath + name + ".md")
 	return string(b), err
 }
+
+func kindString(r *lint.Rule) string {
```

Comment:
s/`kindString`/`ruleKind`?

Reply:
What about `ruleKindString`?
I think this "Kind" string may include more attributes later.
It's like human-readable short overview of a set of rule attributes.
(Because it's dumb to fill columns with true/false..)